src/clj/native_image.clj

```diff
@@ -17,7 +23,8 @@
 
 (defn merged-deps []
   "Merges install, user, local deps.edn maps left-to-right."
-  (-> (deps.reader/clojure-env)
+  (-> (if windows? {:config-files []}
```

Comment:
I'm not the maintainer of this library, but as a maintainer in general I'd be happier if there was at least a reference to the upstream Clojure ticket you mentioned in #6, ideally also a reference to #6 itself.

Reply:
The issue in question is https://clojure.atlassian.net/browse/TDEPS-128